core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -631,5 +633,21 @@ class MapUnit {
         return sum
     }
 
+    private fun getCitadelDamage() {
+        // Check for Citadel damage
+        val applyCitadelDamage = currentTile.neighbors
+                .filter{ it.getOwner() != null && civInfo.isAtWarWith(it.getOwner()!!) }
+                .map{ it.getTileImprovement() }
+                .filter{ it != null && it.hasUnique("Deal 3 damage to adjacent enemy units") }
+                .any()
+
+        if (applyCitadelDamage) {
+            health -= 3
```

Comment:
Probably 30, no?

Reply:
3 in vanilla, 30 in G&K [source](https://civilization.fandom.com/wiki/Citadel_(Civ5))

Reply:
Unciv works with a 100-health point system, so that'll be 30 =)

Reply:
What you think we should do? Keep this to 3 and wait for G&K to update it to 30? 3 seems a really low value, 30 seems a really big one though :thinking: